packages/core-flows/src/region/workflows/update-regions.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  FilterableRegionProps,
+  RegionDTO,
+  UpdatableRegionFields,
+} from "@medusajs/types"
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { updateRegionsStep } from "../steps"
+
+type UpdateRegionsStepInput = {
+  selector: FilterableRegionProps
+  update: UpdatableRegionFields
+}
+
+type WorkflowInput = UpdateRegionsStepInput
+
+export const updateRegionsWorkflowId = "update-regions"
+export const updateRegionsWorkflow = createWorkflow(
```

Comment:
Maybe we can have a simpler approach to running simple CRUD workflows, like some default 
`updateWorkflow("region", selector, update)` that does what you've written here under the hood? This is just an optimization of course, just something to keep in mind as we'll have quite a few simple CRUD operations

Reply:
Love this idea! I imagine we'll find more of these cases when we start tackling workflows more aggresively